### PR TITLE
remove beta-related content from Running Drupal 8 doc

### LINKED ISSUE #909: Running Drupal 8: update content
https://pantheon.io/docs/articles/drupal/running-drupal-8/

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #909

### CODE REVIEW

**Comment by nataliejeremy** [comment]
@ttrowell @alexdicianu Please review. This is in preparation of the D8 launch.

File: source/docs/articles/drupal/running-drupal-8.md [hunk 1]
```diff
@@ -24,22 +20,18 @@ If you have done no modification to Drupal 8 core, this process will work automa
...
-
-## Going Live with Drupal 8
-
-Pantheon currently does not recommend running customer sites in production with Drupal 8. However, if you are doing a community or educational project with Drupal 8 that you would like to go live, please [contact us for a free live Drupal 8 site](https://pantheon.io/free-website-management-platform-beyond-hosting).
+Pantheon supports the [Drupal 8 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `sites/default/config` directory for each Pantheon Drupal 8 site. You can export your configuration into that directory directly using Drush's config-export command or indirectly using Drupal's UI to download the configuration and then use SFTP/Git to place the configuration in `sites/default/config`. For more information on how this all works, check out Matt Cheney and David Strauss' presentation on [Drupal 8 CMI on Managed Workflow at Drupalcon Amsterdam](https://amsterdam2014.drupal.org/session/drupal-8-cmi-managed-workflow).
```
Comment (rachelwhitton) [inline source/docs/articles/drupal/running-drupal-8.md]: Suggest linking to new doc proposed in #921 once it has been created.

File: source/docs/articles/drupal/running-drupal-8.md [hunk 2]
```diff
@@ -28,18 +20,18 @@ If you have done no modification to Drupal 8 core, this process will work automa
...
```
Comment (rachelwhitton) [inline source/docs/articles/drupal/running-drupal-8.md]: Suggest linking to new doc proposed in #921 once it has been created.

**Comment by rachelwhitton** [comment]
Should we link to new doc in #936 somewhere here? Thoughts?

File: source/docs/articles/drupal/running-drupal-8.md (hunk 2 above)
Comment (bmackinney) [inline source/docs/articles/drupal/running-drupal-8.md]: :+1:

**Comment by bmackinney** [comment]
Yes, @rachelwhitton, link to both new docs.

**Comment by rachelwhitton** [comment]
This doc is repurposed/deleted with info preserved in #999

**Pull request closed without merging**